packages/modular-scripts/src/build/buildPackage/makeBundle.ts

```diff
@@ -107,7 +108,7 @@ export async function makeBundle(
 
   const outputOptions: rollup.OutputOptions = {
     freeze: false,
-    sourcemap: true, // TODO: read this off env
+    sourcemap: getConfig('generateSourceMap') as boolean,
```

Comment:
The `as boolean` is safe, as we know for sure what getConfig for each configuration field will return